apps/betterangels-backend/common/models.py

```diff
@@ -220,6 +221,25 @@ def get_or_create_location(cls, location_data: Dict[str, Any]) -> "Location":
         return location
 
 
+class PhoneNumber(models.Model):
+    number = PhoneNumberField(region="US", blank=True, null=True)
+    is_primary = models.BooleanField(default=False)
+
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE)
+    object_id = models.PositiveIntegerField()
+    content_object: GenericForeignKey = GenericForeignKey("content_type", "object_id")
```

Comment:
Is this used automatically?

Reply:
not sure I understand the question. Automatically as opposed to needing to be defined/invoked somewhere?

Reply:
Related to previous question, if generic foreign keys create indices then the previous recommendation isn't necessary.

Reply:
See: https://docs.djangoproject.com/en/5.1/ref/contrib/contenttypes/#generic-relations

Reply:
ya, there's no index created. I updated that